Upstream-Status: Inapproriate

This patch is for ptest, so it is inapproriate to send to upstream.

Update context for 1.3.5.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
From 0780181a3285511f166487a54ebc231fc657edfe Mon Sep 17 00:00:00 2001
From: Catalin Enache <catalin.enache@windriver.com>
Date: Mon, 25 Jul 2016 16:38:51 +0300
Subject: [PATCH] Install missing conf file

openvzutilstest.conf file is needed by openvzutilstest test.

Signed-off-by: Catalin Enache <catalin.enache@windriver.com>
---
 tests/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 02e0dd8..187fbca 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -170,6 +170,7 @@ EXTRA_DIST =		\
 	xml2vmxdata
 
 test_helpers = commandhelper ssh virconftest
+test_misc =
 test_programs = virshtest sockettest \
 	nodeinfotest virbuftest \
 	commandtest seclabeltest \
@@ -259,6 +260,7 @@ endif WITH_LXC
 
 if WITH_OPENVZ
 test_programs += openvzutilstest
+test_misc += openvzutilstest.conf
 endif WITH_OPENVZ
 
 if WITH_ESX
@@ -1293,7 +1295,7 @@ endif ! WITH_CIL
 
 buildtest-TESTS: $(TESTS) $(test_libraries) $(test_helpers)
 
-PTESTS = $(TESTS) $(test_helpers) test-lib.sh schematestutils.sh
+PTESTS = $(TESTS) $(test_helpers) $(test_misc) test-lib.sh schematestutils.sh
 
 install-ptest:
 	list='$(TESTS) $(test_helpers) test-lib.sh schematestutils.sh'
-- 
2.7.4

